app/src/test/java/com/kickstarter/ui/activities/compose/search/FilterMenuBottomSheetTest.kt

```diff
@@ -51,32 +53,33 @@ class FilterMenuBottomSheetTest : KSRobolectricTestCase() {
     }
 
-// TODO failing an assert for unknown reasons, when running it with breakpoints never reached the last iteration for availableFilters,
-// TODO when running the real app with breakpoints always reaches the last iteration for availableFilters quite likely test setup issue but the love of god cannot find the issue
-
-//    @Test
-//    fun `test FilterMenuBottomSheet renders all available filter Rows with ffOn`() {
-//        val shouldShowPhase = true
-//        composeTestRule.setContent {
-//            KSTheme {
-//                FilterMenuBottomSheet(
-//                    modifier = Modifier.width(500.dp).height(800.dp),
-//                    selectedProjectStatus = DiscoveryParams.State.LIVE,
-//                    onApply = {a, b -> },
-//                    onDismiss = {},
-//                    onNavigate = {},
-//                    availableFilters = if (shouldShowPhase) FilterType.values().asList()
-//                    else FilterType.values().asList().filter { it != FilterType.PERCENTAGE_RAISED }
-//                )
-//            }
-//        }
-//
-//        composeTestRule.onNodeWithTag(FilterMenuTestTags.SHEET).assertIsDisplayed()
-//        composeTestRule.onNodeWithTag(FilterMenuTestTags.CATEGORY_ROW).assertIsDisplayed()
-//        composeTestRule.onNodeWithTag(FilterMenuTestTags.PROJECT_STATUS_ROW).assertIsDisplayed()
-//        composeTestRule.onNodeWithText(FilterMenuTestTags.PERCENTAGE_RAISED_ROW).assertExists()
-//        composeTestRule.onNodeWithTag(FilterMenuTestTags.PERCENTAGE_RAISED_ROW).assertIsDisplayed()
-//    }
+    @Test
+    fun `test FilterMenuBottomSheet renders all available filter Rows with ffOn`() {
+        composeTestRule.setContent {
+            val shouldShowPhase = true
+            KSTheme {
+                FilterMenuBottomSheet(
+                    selectedProjectStatus = DiscoveryParams.State.LIVE,
+                    onApply = { a, b -> },
+                    onDismiss = {},
+                    onNavigate = {},
+                    availableFilters = if (shouldShowPhase) FilterType.values().asList()
+                    else FilterType.values().asList().filter { it != FilterType.PERCENTAGE_RAISED }
+                )
+            }
+        }
+
+        composeTestRule.onNodeWithTag(FilterMenuTestTags.SHEET).assertIsDisplayed()
+        composeTestRule.onNodeWithTag(FilterMenuTestTags.CATEGORY_ROW).assertIsDisplayed()
+        composeTestRule.onNodeWithTag(FilterMenuTestTags.PROJECT_STATUS_ROW).assertIsDisplayed()
+
+        // - As working with LazyColumns, not all elements of the list are composed until the elements is visible
+        // - perform a scroll on the list, to reach the desired not, once scroll performed THEN the element will be composed and added to the semantic tree
+        composeTestRule
+            .onNodeWithTag(FilterMenuTestTags.LIST)
+            .performScrollToNode(hasTestTag(FilterMenuTestTags.PERCENTAGE_RAISED_ROW))
```

Comment:
Basically as the comment highlights, when dealing with `LazyColum` not all items are composed (added to the semantic tree), from the fist load, it requires for the item to be visible, so in order to test % raised new filter (which is the third element), required scrolling for it to become visible